conviva/src/main/java/com/bitmovin/analytics/conviva/ContentMetadataBuilder.java

```diff
@@ -43,44 +44,53 @@ public void setPlaybackStarted(boolean value) {
         playbackStarted = value;
     }
 
-    public ContentMetadata build() {
+    public Map<String, Object> build() {
+
         if (!playbackStarted) {
-            // Asset name is only allowed to be set once
-            if (contentMetadata.assetName == null) {
-                contentMetadata.assetName = getAssetName();
+            if(!contentInfo.containsKey(ConvivaSdkConstants.ASSET_NAME)) {
+                contentInfo.put(ConvivaSdkConstants.ASSET_NAME, getAssetName());
             }
 
-            contentMetadata.viewerId = getViewerId();
-            contentMetadata.streamType = ObjectUtils.defaultIfNull(
+            contentInfo.put(ConvivaSdkConstants.VIEWER_ID, getViewerId());
+
+            ConvivaSdkConstants.StreamType streamType = ObjectUtils.defaultIfNull(
                     metadataOverrides.getStreamType(),
                     metadata.getStreamType());
+            boolean isLive = streamType == ConvivaSdkConstants.StreamType.LIVE;
+            contentInfo.put(ConvivaSdkConstants.IS_LIVE, isLive);
```

Comment:
`ConvivaSdkConstants.IS_LIVE` is not set in original code, any specific reason to add it now?